src/System.Private.ServiceModel/tests/Scenarios/SelfHostWcfService/WcfService.csproj

```diff
@@ -5,12 +5,27 @@
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <ProjectGuid>{CBE9909F-4CF7-4AD0-9307-BD0DFBB1105E}</ProjectGuid>
-    <OutputType>Exe</OutputType>
+    <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>WcfService</RootNamespace>
     <AssemblyName>WcfService</AssemblyName>
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
+    <PublishUrl>publish\</PublishUrl>
+    <Install>true</Install>
+    <InstallFrom>Disk</InstallFrom>
+    <UpdateEnabled>false</UpdateEnabled>
+    <UpdateMode>Foreground</UpdateMode>
+    <UpdateInterval>7</UpdateInterval>
+    <UpdateIntervalUnits>Days</UpdateIntervalUnits>
+    <UpdatePeriodically>false</UpdatePeriodically>
+    <UpdateRequired>false</UpdateRequired>
+    <MapFileExtensions>true</MapFileExtensions>
+    <ApplicationRevision>0</ApplicationRevision>
+    <ApplicationVersion>1.0.0.%2a</ApplicationVersion>
+    <IsWebBootstrapper>false</IsWebBootstrapper>
+    <UseApplicationTrust>false</UseApplicationTrust>
+    <BootstrapperEnabled>true</BootstrapperEnabled>
```

Comment:
Interesting. VS must have added them. 

-----Original Message-----
From: "Jason Pang" <notifications@github.com>
Sent: ‎6/‎24/‎2015 1:24 AM
To: "dotnet/wcf" <wcf@noreply.github.com>
Cc: "Dustin Metzgar" <dustin.metzgar@gmail.com>
Subject: Re: [wcf] Bridge test changes (#115)

In src/System.Private.ServiceModel/tests/Scenarios/SelfHostWcfService/WcfService.csproj:
>      <AppDesignerFolder>Properties</AppDesignerFolder>
>      <RootNamespace>WcfService</RootNamespace>
>      <AssemblyName>WcfService</AssemblyName>
>      <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
>      <FileAlignment>512</FileAlignment>
> +    <PublishUrl>publish\</PublishUrl>
> +    <Install>true</Install>
> +    <InstallFrom>Disk</InstallFrom>
> +    <UpdateEnabled>false</UpdateEnabled>
> +    <UpdateMode>Foreground</UpdateMode>
> +    <UpdateInterval>7</UpdateInterval>
> +    <UpdateIntervalUnits>Days</UpdateIntervalUnits>
> +    <UpdatePeriodically>false</UpdatePeriodically>
> +    <UpdateRequired>false</UpdateRequired>
> +    <MapFileExtensions>true</MapFileExtensions>
> +    <ApplicationRevision>0</ApplicationRevision>
> +    <ApplicationVersion>1.0.0.%2a</ApplicationVersion>
> +    <IsWebBootstrapper>false</IsWebBootstrapper>
> +    <UseApplicationTrust>false</UseApplicationTrust>
> +    <BootstrapperEnabled>true</BootstrapperEnabled>
What are these changes from? 
—
Reply to this email directly or view it on GitHub.